src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -248,57 +248,6 @@ winrt::Microsoft::Terminal::Settings::Model::CascadiaSettings CascadiaSettings::
         // If this throws, the app will catch it and use the default settings
         resultPtr->_ValidateSettings();
 
-        // GH 3855 - Gathering Data on custom profiles to inform better defaults

```

Comment:
Covered this with Dustin. This was a quick-n-easy way to gather data on the key bindings. Since we removed `KeyMapping`, we don't actually have a way to serialize our key bindings (yet).

Makes more sense to just outright remove this, then we leverage the settings model to gather data better later (if desired).